### ISSUE TITLE
Pomodoro Clock #15 Test of "25 + 5" Test Suite May Be Yielding Incorrect Results

### ISSUE THREAD

**Issue by noelbray** [freeCodeCamp/testable-projects-fcc#1228]
#### Issue Description
(If I'm posting this in the wrong place, I'm sorry. But, I was told that if I had still had issues, to go to github, so I am.)

My Pomodoro Clock is functioning fine, but is failing #15 test in the test. 

Here is part of what it's yielding: 

"15. When a break countdown reaches zero (NOTE: timer MUST reach 00:00), a new session countdown should begin, counting down from the value currently displayed in the id="session-length" element.
Cannot read property '1' of null
TypeError: Cannot read property '1' of null
"

I clicked “debugger mode” in my Codepen, and then I opened the console and it doesn’t show any errors even when I’m running my clock or doing different things with it.

Someone believed it had to do with the pause() method, but I don't believe that is the case because I commented out the pause method code as well as the currentTime, play, and other related code to see what would happen and it still returns the same result as above.  (since I commented those out, the test related to them fail which is normal, just look at 15.)

I found out the error is happening with the sessionLength variable/value of the minutes key/property in the setSessionTimer function by changing it from sessionLength to 1 and 25 which slightly changed the results of the 15th test.

I also used console.log right after that line of code to see what, if anything, the sessionLength variable is yielding, as well as to see what some other things are yielding too. And they all were yielding the correct values in the console.

Here is the part of code the above is referring too: 
```
   if (sessionTimer.minutes == 0 && sessionTimer.seconds == 0) {
                // beep.play();
                setSessionTimer(prev => ({minutes: sessionLength, seconds: "00"}));
                console.log("Session Timer Error Point", sessionTimer.minutes, sessionTimer.seconds, sessionLength, SessionLength, {minutes: sessionLength, seconds: "00"});
                
                setTimerLabel("Break Timer");
              }
```

Here's my Codepen: 

https://codepen.io/nwbnwb/full/VwaWvPR

**Comment by KartikSoneji** [comment]
Hi @noelbray 

This might be an issue with your setup, since I am unable to reproduce the error myself:

Neither on CodePen
![image](https://user-images.githubusercontent.com/25801283/92813624-2b6c5d00-f3df-11ea-9bcb-a1c5bc204307.png)

Nor locally (on CodePen, click `Export` > `Export .zip`)
![image](https://user-images.githubusercontent.com/25801283/92813509-e9dbb200-f3de-11ea-9053-e7973b1384ac.png)

Can you please post the name and exact version of the browser you are using?

Also, try running the tests in Guest Mode or Incognito Mode, and see if the results differ.
If they do, then one of your extensions might be interfering with the tests.

**Comment by noelbray** [comment]
@KartikSoneji Thank you for checking that for me. In the FCC forum, another guy and I, mainly him, recently figured out that when my countdown timers were switching from the Break Timer to the Session Timer, it was not prepending a "0" to numbers <= 9. I though I accounted for all cases, but apparently in the switch it wasn't prepending "0". Despite my debugging efforts, I didn't find out, catch it.

I feel horrible for creating a bug report, inquiry. 

I'm sorry. I'm doing my best to put everything I'm teaching myself (through free resources) together and trying to debug.

Once again, I'm sorry, and I'm sorry if I wasted some of your time. 

If you want to remove this report feel free to. Or if you want me to, just let me know how, and I will. 

I hope you have a great day. Keep smiling, and keep up the good work. : )
